### DISCUSSION TITLE
Can't upload .ipynb file as gist

### DISCUSSION THREAD

**Discussion by phenders** [community/community#23782] [New to GitHub]
<p>Whenever I drag/drop or copy/paste an iPython notebook .ipynb file into <a href="http://gist.github.com" rel="nofollow noopener">gist.github.com</a> I get a “Sorry, something went wrong. Reload?” message. This has been working fine for me in the past up until last week but just started failing today.</p>

**Comment by phenders** [comment, marked as answer]
<p>The problem is now fixed.</p>

**Comment by ModMaamari** [comment]
<p>I still have the same problem , how did you solve it ?</p>

**Comment by DanielGoldfarb** [comment]
<p>To anyone who works for github:</p>
<p>I have been grappling with the  <code>Sorry, something went wrong. [Reload?]</code>  issue for months. I have many notebooks in many repositories. (<a href="https://github.com/matplotlib/mplfinance/tree/master/examples" rel="nofollow noopener">One such example is here</a>).</p>
<p>Whenever the notebooks fail with  <code>Sorry, something went wrong. [Reload?]</code>  ,  <em>the same notebook will be fine some time later</em>  , where “ <em>some time later</em> ” can be anywhere from several minutes to several hours (up to maybe a day). However that doesn’t mean that it won’t start failing again several hours after that.</p>
<p>I am convinced  <strong><a href="http://github.com/" rel="nofollow noopener">github.com</a></strong>  is using a service (hosted somewhere) to render notebooks, and that  <em>sometimes this service either gets into a bad state, or crashes; when that happens notebooks won’t render until the service is bounced (or somehow has its “good” state restored).</em></p>
<p>I would be happy to volunteer my time to help investigate the issue, if it is otherwise not a priority for <a href="http://github.com/" rel="nofollow noopener">github.com</a> to allocate the resources. Please let me know if there is anything I can do to help.</p>
<p>This can be very inconvenient, even embarrassing, to say the least, when I am trying to provide support to users of my public repositories and they tell me that the notebooks to which I direct them are not rendering.</p>
<p>Thank you.</p>
